scm-webapp/src/main/java/sonia/scm/security/JwtAccessTokenBuilder.java

```diff
@@ -183,8 +182,8 @@ public JwtAccessToken build() {
 
 
     if (refreshableFor > 0) {
-      long refreshExpiration = refreshableForUnit.toMillis(refreshableFor);
-      claims.put(JwtAccessToken.REFRESHABLE_UNTIL_CLAIM_KEY, new Date(now.toEpochMilli() + refreshExpiration).getTime());
+      long re = refreshableForUnit.toMillis(refreshableFor);
+      claims.put(JwtAccessToken.REFRESHABLE_UNTIL_CLAIM_KEY, new Date(now.toEpochMilli() + re).getTime());
```

Comment:
What do we do here? Get millis, do some calculation, create a date, just to get the millis again? I don't get it.
```suggestion
      claims.put(JwtAccessToken.REFRESHABLE_UNTIL_CLAIM_KEY, now.toEpochMilli() + re);
```
It looks like we don't have a test for this :-(

Reply:
I've seen that the claim supports date, so i've used a slightly different way and i added some tests too.